src/libraries/Microsoft.Extensions.Hosting.WindowsServices/src/WindowsServiceLifetime.cs

```diff
@@ -62,6 +62,9 @@ public WindowsServiceLifetime(IHostEnvironment environment, IHostApplicationLife
         private IHostEnvironment Environment { get; }
         private ILogger Logger { get; }
 
+        /// <summary>Asynchronously waits until start is complete before continuing. This method is called at the beginning of <see cref="Microsoft.Extensions.Hosting.IHost.StartAsync(System.Threading.CancellationToken)" />. This can be used to delay startup until signaled by an external event.</summary>
```

Comment:
```suggestion
        /// <summary>Asynchronously waits until start is complete before continuing. This method is called at the beginning of <see cref="IHost.StartAsync(CancellationToken)" />. This can be used to delay startup until signaled by an external event.</summary>
```